scrapers/GroobyNetwork-Partial.yml

```
@@ -18,11 +18,12 @@ sceneByURL:
       - tgirls.xxx
       - transexpov.com
       - transgasm.com
+      - tgirls.porn
     scraper: sceneScraper
 xPathScrapers:
   sceneScraper:
     scene:
-      Title: //p[@class="trailertitle"]/text()
+      Title: //div[@class="trailer_toptitle_left"]/text()
```

This breaks sites that have titles with performer anchors
The suggested below uses the existing  selector for all sites except tgirls.porn where yours is used
```suggestion
      Title: //p[@class="trailertitle"]/text()|//div[@class="trailer_toptitle_left"]/text()
```